database/sqlite/sqlite_aclk_alert.c

```diff
@@ -685,8 +687,8 @@ void aclk_start_alert_streaming(char *node_id, bool resets)
 }
 
 #define SQL_QUEUE_REMOVE_ALERTS "INSERT INTO aclk_alert_%s (alert_unique_id, date_created, filtered_alert_unique_id) " \
-                                "SELECT unique_id alert_unique_id, UNIXEPOCH(), unique_id alert_unique_id FROM health_log_%s " \
-                                "WHERE new_status = -2 AND updated_by_id = 0 AND unique_id NOT IN " \
+                                "SELECT unique_id alert_unique_id, UNIXEPOCH(), unique_id alert_unique_id FROM health_log " \
```

Comment:
In previous query we had `unixepoch()` in lowercase, here we have in uppercase, I suggest to use only one pattern for functions.